editor/utils/block-list.js

```diff
@@ -32,34 +41,94 @@ const INNER_BLOCK_LIST_CACHE = {};
  */
 export function createInnerBlockList( uid, renderBlockMenu, showContextualToolbar ) {
 	if ( ! INNER_BLOCK_LIST_CACHE[ uid ] ) {
-		INNER_BLOCK_LIST_CACHE[ uid ] = [
-			// The component class:
-			class extends Component {
-				componentWillMount() {
-					INNER_BLOCK_LIST_CACHE[ uid ][ 1 ]++;
+		const InnerBlockListComponent = class extends Component {
+			constructor() {
+				super( ...arguments );
+				this.updateNestedSettings = this.updateNestedSettings.bind( this );
+				this.insertTemplateBlocks = this.insertTemplateBlocks.bind( this );
```

Comment:
We shouldn't need to bind these two methods since they aren't being used as a callback anywhere, e.g. they aren't passed as an `on[Event]` prop.